lib/screens/home.dart

```diff
@@ -166,19 +167,23 @@ class _HomeScreenState extends State<HomeScreen>
                                         onCapture: (data) async {
                                           // print("Sacnned:: $data");
                                           // print("Sacnned len:: ${data.length}");
-                                          if (data.length > 64) {
+                                          String err =
+                                              "${validation.did(data)}";
+                                          if (err.isNotEmpty) {
                                             _chargeProvider.setStatus(
                                                 LoadingStatus.error,
-                                                message:
-                                                    Env.invalidProviderDid);
-
+                                                message: err);
                                             return;
                                           }
                                           _dumbChargeProvider!.qrController
                                               .pause();
                                           _chargeProvider.providerDid = data;
+                                          _chargeProvider
+                                              .generateAndFundMultisigWallet();
+
                                           await _chargeProvider
                                               .fetchProviderDidDocument(data);
+                                          if (!mounted) return;
```

Comment:
I didn't see any other place using this parameter. Do I miss something?

Reply:
It's a flutter magic :) 
`mounted` is actually a flutter global scope variable that's inferred.

The code is saying:
if the the screen is not yet mounted, return.